chromium_src/build/android/gyp/lint.py

```diff
@@ -15,3 +14,21 @@ def JavaCmd(original_function, xmx='1G'):  #NOSONAR
     # for Android incremental builds
 
     return original_function('4G')
+
+
+@override_utils.override_function(globals())
+def _GenerateConfigXmlTree(original_function, orig_config_path,
+                           backported_methods):
+    # Expand upstream's lint-suppressions.xml with python chromium_src override
+    original_root_node = original_function(orig_config_path,
+                                           backported_methods)
+
+    if orig_config_path:
+        brave_config_path = os.path.join(os.pardir, os.pardir, 'brave',
+                                         'android', 'expectations',
+                                         'lint-suppressions.xml')
```

Comment:
let's modify few things to make it work as other `chromium_src` overrides (a rebuild should happen when an override is modified):

1. the override file should be placed at `//brave/chromium_src/...` path.
2. this call should be replaced with `brave_chromium_utils.get_chromium_src_override(orig_config_path)`.
3. `touchOverriddenFiles` in `brave/build/commands/lib/util.js` should be extended to support `.xml` files (there's a list of extensions to handle).

Reply:
@goodov , thanks for the comments, they improved rebuild experience indeed

fixed at https://github.com/brave/brave-core/pull/27020/commits/c9ba6986f5cfad17820b9393dfe7997f47d77a16